webui/react/src/components/Table/InteractiveTable.tsx

```diff
@@ -662,6 +665,7 @@ const InteractiveTable = <
             bordered
             columns={renderColumns}
             components={components}
+            data-testid="table"
```

Comment:
it should be fine because this table is nested underneath another `data-test-component="interactiveTable"`. If there were a second table, we'd expect it to live under a different hierarchy. Even if we had two interactiveTables, each selector could look like this
```js
[data-test-component="interactiveTable"][data-testid="tableA"]
[data-test-component="interactiveTable"][data-testid="tableB"]
```
and so the underlying tables would still be separate.

Reply:
gotcha!